spec/policies/agent/rdv_policy_spec.rb

```diff
@@ -136,5 +136,24 @@
     end
   end
 
+  context "any participating user is soft deleted" do
```

Comment:
peut-être renommer ça pour signifier que tu testes aussi si un agent est soft-deleted ? 

```suggestion
  context "a participating user and a participating agent are soft deleted" do
```